### PR TITLE
if the glob is not present at all, a FileNotFoundException is raised. Ca...

### PR DESCRIPTION
**Pull request by plinton**
...tch that exception and continue which should allow execution to continue.

The goal is to have RangeHourly continue and eventually make the glob once it has finished.

### CODE REVIEW

**Comment by ulzha** [comment]
Yeah, limiting to snakebite isn't intended. Checking exists() would be good.

File: test/range_test.py [hunk 1]
```diff
@@ -325,6 +332,7 @@ def test_successfully_inferred(self):
         ])
 
     @mock.patch('luigi.mock.MockFileSystem.listdir', new=mock_listdir)  # fishy to mock the mock, but MockFileSystem doesn't support globs yet
+    @mock.patch('luigi.mock.MockFileSystem.exists', new=mock_listdir)
```
Comment (Tarrasch) [inline test/range_test.py]: You mock exists with listdir? Looks fishy :)

**Comment by Tarrasch** [comment]
Can you reword the commit message? It violates 50 character length [1], and it doesn't start with describing that it solves a bug for `tools.range`.

[1]: http://stopwritingramblingcommitmessages.com/

**Comment by plinton** [comment]
@ulzha @Tarrasch @erikbern 
Updated with the exists check, better commit message, tests that actually check the right thing and a little documentation.

**Comment by erikbern** [comment]
LGTM

**Comment by plinton** [comment]
I don't have merge permission so @erikbern, @Tarrasch or @ulzha can you merge please?

**Pull request merged**